backend/app/users/models.py

```diff
@@ -82,8 +84,8 @@ async def save_user_to_db(
 
     user_db = UserDB(
         username=user.username,
-        experiments_quota=user.experiments_quota,
-        api_daily_quota=user.api_daily_quota,
+        experiments_quota=DEFAULT_EXPERIMENTS_QUOTA,
```

Comment:
In the pydantic model, `UserCreate` we are sending experiments_quota and api_daily_quota. Here we are not using it. Should this instead be `user.experiments_quota or DEFAULT_EXPERIMENTS_QUOTA` or similar?

We were originally making the assumption that if this is not provided then these quotas as infinite. We can fix this later when we actually enable these